webapp/channels/src/components/login/login.tsx

```diff
@@ -674,7 +685,9 @@ const Login = ({onCustomizeHeader}: LoginProps) => {
 
     const onEnterKeyDown = (e: React.KeyboardEvent<HTMLInputElement>) => {
         if (e.key === Constants.KeyCodes.ENTER[0]) {
-            preSubmit(e);
+            if(!isExternalBtnFocused){
```

Comment:
shouldn't this be a positive evaluation? I mean, the check should be `if (isExternalBtnFocused)`